.travis.yml

```diff
@@ -36,6 +36,10 @@ matrix:
       env: DJANGO="django>=1.10.0,<1.11.0" REST_FRAMEWORK="djangorestframework>=3.4,<3.5"
     - python: "3.3"
       env: DJANGO="django>=1.9.0,<1.10.0" REST_FRAMEWORK="djangorestframework>=3.4,<3.5"
+  include:
+    - python: "3.5"
+      env: W='python -W error -m' DJANGO='https://github.com/django/django/archive/master.tar.gz' REST_FRAMEWORK='djangorestframework>=3.4,<3.5'
   allow_failures:
+    - env: W='python -W error -m' DJANGO='https://github.com/django/django/archive/master.tar.gz' REST_FRAMEWORK='djangorestframework>=3.4,<3.5'
```

Comment:
Either way, the root problem is that the warnings are being raised during the VM's initialization, not during testing.